src/plugins/vis_type_xy/public/vis_renderer.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { lazy } from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+
+import { I18nProvider } from '@kbn/i18n/react';
+
+import { ExpressionRenderDefinition } from '../../expressions/public';
+import { VisualizationContainer } from '../../visualizations/public';
+
+import { XyVisType } from '../common';
+import { SplitChartWarning } from './components';
+import { VisComponentType } from './vis_component';
+import { RenderValue, visName } from './xy_vis_fn';
+
+// @ts-ignore
+const VisComponent = lazy<VisComponentType>(() => import('./vis_component'));
+
+function shouldShowNoResultsMessage(visData: any, visType: XyVisType): boolean {
+  if (['goal', 'gauge'].includes(visType)) {
+    return false;
```

Comment:
@nickofthyme I think this is not needed here, right?

Reply:
Yes thank you. See cab7b10cc49482306a9d9de531a2f5a4e912d172